### ISSUE TITLE
Custom training data has poor quality on hand

### ISSUE THREAD

**Issue by ch1998** [hustvl/4DGaussians#31]
I started from  images using the enerf-outdoor dataset ，used colmap to generate camera parameters and init point clouds. The background quality in the result is very good, but once the characters move there is a noticeable loss of quality.
look at face and hand
![00106](https://github.com/hustvl/4DGaussians/assets/37291486/301f0ae0-5610-403b-a2bb-b332a61671f2)

**Comment by guanjunwu** [comment]
maybe you can enlarge the batch size and make the training process longer?
now I also face the same problem and I'm working hard to solve it QwQ

**Comment by Harryhaozhu** [comment]
> 我首先使用 enerf-outdoor 数据集中的图像，然后用 colmap 生成相机参数并初始化点云。结果的背景质量非常好，但一旦人物移动，质量就会明显下降。 请看脸部和手部。 ![00106](https://github.com/hustvl/4DGaussians/assets/37291486/301f0ae0-5610-403b-a2bb-b332a61671f2)

请问您是如何生成.npy的，我使用llff的imgs2poses.py生成的.npy与dynerf数据集提供的.npy差异很大，导致训练一塌糊涂
